### ISSUE TITLE
Not working in mobile Android browser

### ISSUE THREAD

**Issue by karsunke** [bevacqua/horsey#36]
I tried horsey on several devices. It works in every Desktop browser I tested. Also on iOS browsers. But there seems to be a problem with the mobile Android browser (Chrome).

Typing in the box does not fire any events. No list is shown. I debugged it and there is no arrow message. You can test it at the sample page http://bevacqua.github.io/horsey/

I think this is a major bug. It stops me from using the library.

**Comment by karsunke** [comment]
I managed to find a workaround. Just add a keydown event listener and call the methods hide and show. With jQuery it looks like this:

    `var jElement = $("#address-search");
    jElement.keydown(function() {
        horseyElement.hide();
        horseyElement.show();
    })`

**Comment by bevacqua** [comment]
Is this still the case?

**Comment by karsunke** [comment]
Yep. Still the same. I tried it again with the latest Android Chrome Browser.

**Comment by akrawchyk** [comment]
@karsunke thanks for the workaround!

**Comment by akrawchyk** [comment]
@karsunke this does the same except maintains the user's ability to navigate the list via keyboard arrows:

```
var KEY_UP = 38;
var KEY_DOWN = 40;
$('#{{ input_id }}').keyup(function(e) {
  if (!e.keyCode === KEY_UP && !e.keyCode === KEY_DOWN) {
    horseyInstance.hide();
    horseyInstance.show();
  }
});
```

**Comment by karsunke** [comment]
@akrawchyk Thanks for the addition.

**Comment by zspitzer** [comment]
Alas, Chrome is pretty pain with keydown events and soft keyboards, here is the WONTFIX bug from 2012 with 259 comments! https://bugs.chromium.org/p/chromium/issues/detail?id=118639

Solution might to use the newer "input" rather than keydown event which is supported by everything since IE9
https://developer.mozilla.org/en/docs/Web/API/GlobalEventHandlers/oninput

**Comment by beatfreaker** [comment]
Hi @karsunke i am facing the same issue can you please help me out. I tried your approach but i didn't got what `horseyElement` should be.

can you please give me an example?

```
var horseyElement = horsey(document.querySelector('#profile-cities'), {
	source: function(data, done) {;
	}
});
document.querySelector('#profile-cities').keydown(function() {
    horseyElement.hide();
    horseyElement.show();
});
```

this is my code, can you please correct the code and tell me where i am going wrong?
